JS-Week-3/SmartPhoneUsage.js

```diff
@@ -0,0 +1,66 @@
+const activities = [];
+
+
+function addActivity(date, activity, duration) {
+    const activityObject = {
+        date: new Date().toLocaleDateString(),
+        activity: activity,
+        duration: duration
+    };
+    activities.push(activityObject);
+}
+addActivity("23/7-18", "Youtube", 30);
+console.log(activities);
+
+let usageLimit = 100;
+function showStatus() {
+    if (activities.length === 0) {
+        return "Add some activities before calling showStatus";
+    }
+    
+    let totalActivities = activities.length;
+    let totalDuration = activities.reduce((sum, activity) => sum + activity.duration, 0);
+    
+    return `You have added ${totalActivities} activities. They amount to ${totalDuration} min. of usage`;
+}
+
+
+console.log(showStatus()); 
+ 
+
+// Modified showStatus function with limit check
+function showStatus() {
```

Comment:
It would have been better to keep only one function with that name in the file instead of repeating some of the code twice.